antora.yml

```diff
@@ -3,9 +3,12 @@ title: Bonita
 version: 2021.1
 asciidoc:
   attributes:
-    varVersion: 2021.1
-    bonitaTechnicalVersion: '7.12.1'
+    bonitaVersion: 2021.1
+    bonitaTechnicalVersion: '7.12.1' # latest public community version
```

Comment:
```suggestion
    bonitaTechnicalVersion: '7.12.1' # latest public community version on this X.Y version line
```

Reply:
I can add this information in the comment
But shouldn't I rename the attribute `bonitaCommunityTechnicalVersion` into to make it more explicit to remove the comment? Like I tried to do for `bonitaDockerImageEnterpriseVersion`

Reply:
I think not, as I think this version is also used in Enterprise code samples, even if for Enterprise, there may be more recent maintenance versions